wallets/index.js

```diff
@@ -77,17 +77,7 @@ export function useWallet (name) {
   }, [wallet, config, toaster])
 
   const save = useCallback(async (newConfig) => {
-    // testConnectClient should log custom INFO and OK message
-    // testConnectClient is optional since validation might happen during save on server
-    // TODO: add timeout
-    let validConfig
-    try {
-      validConfig = await wallet.testConnectClient?.(newConfig, { me, logger })
-    } catch (err) {
-      logger.error(err.message)
-      throw err
-    }
-    await saveConfig(validConfig ?? newConfig, { logger })
```

Comment:
This ran always if `testConnectClient` existed even if no client config was specified.

Moving it into the if block where the client config actually gets saved fixes that.